firebase-messaging.html

```diff
@@ -249,7 +255,7 @@
 
         _bootstrapApp: function(app, customSw) {
           if (app && !customSw) {
-            this.activate();
+            this.activate(null);
```

Comment:
this exposes a problem that the user will no longer be able to simply call `this.activate` in closure. Please change the `@param` in activate to have `@param{ServiceWorkerRegistration|undefined}`